command.go

```diff
@@ -640,6 +640,95 @@ func (cmd *FloatCmd) readReply(rd *proto.Reader) error {
 	cmd.val, cmd.err = rd.ReadFloatReply()
 	return cmd.err
 }
+//------------------------------------------------------------------------------
+type SlowLog struct {
+	Id       int64
+	ExecTime int64
+	Time     int64
+	Args     []string
+}
+
+type SlowLogCmd struct {
+	baseCmd
+	val []*SlowLog
+}
+
+var _ Cmder = (*SlowLogCmd)(nil)
+
+func NewSlowLogCmd(args ...interface{}) *SlowLogCmd {
+	return &SlowLogCmd{
+		baseCmd: baseCmd{_args: args},
+	}
+}
+
+func (cmd *SlowLogCmd) readReply(rd *proto.Reader) error {
+	var v interface{}
+	v, cmd.err = rd.ReadArrayReply(commandSlowLogParser)
+	if cmd.err != nil {
+		return cmd.err
+	}
+	cmd.val = v.([]*SlowLog)
+	return nil
+}
+
+func (cmd *SlowLogCmd) Result() ([]*SlowLog, error) {
+	return cmd.val, cmd.err
+}
+
+// Implements proto.MultiBulkParse
+func commandSlowLogParser(rd *proto.Reader, n int64) (interface{}, error) {
+	logs := make([]*SlowLog, 0, n)
+	for i := int64(0); i < n; i++ {
+		v, err := rd.ReadReply(slowLogParse)
+		if err != nil {
+			return nil, err
+		}
+		vv := v.(*SlowLog)
+		logs =  append(logs, vv)
+	}
+	return logs, nil
+}
+
+func slowLogParse(rd *proto.Reader, n int64) (interface{}, error) {
+	var slowLog SlowLog
+	id, err := readString2Int(rd)
```

Comment:
Is it possible to use rd.ReadInt?

Reply:
At first,  I thought so. but when i use `rd.ReadInt` an error occured
```
redis: can't parse string reply: ":1544754674"
```

so I changed to `rd.ReadString`

Reply:
Then you shoud use ReadIntReply https://github.com/go-redis/redis/blob/master/internal/proto/reader.go#L90